listenbrainz/domain/lastfm.py

```diff
@@ -0,0 +1,119 @@
+import uuid
+
+import requests
+from flask import current_app
+from psycopg2.extras import execute_values
+from requests.adapters import HTTPAdapter, Retry
+from sqlalchemy import text
+
+from listenbrainz import db
+from brainzutils import musicbrainz_db
+
+from listenbrainz.webserver.errors import APINotFound
+
+
+def bulk_insert_loved_tracks(user_id: int, feedback: list[tuple[int, str]]):
+    # DO NOTHING because we can only import loved tracks.
+    # if a user has imported something before from last.fm then the score doesn't change.
+    # if the user has unloved a track on last.fm, we won't receive it this time so can't
+    # remove it from feedback in any case.
+    query = """
+        INSERT INTO recording_feedback (user_id, created, recording_mbid, score)
+             VALUES %s
+        ON CONFLICT (user_id, recording_mbid)
+         DO NOTHING
+    """
+    connection = db.engine.raw_connection()
+    with connection.cursor() as cursor:
+        execute_values(cursor, query, feedback, template=f"({user_id}, to_timestamp(%s), %s, 1)")
+    connection.commit()
+
+
+def load_recordings_from_tracks(track_mbids: list) -> dict[str, str]:
```

Comment:
While this function is rather simple and well named, its existence isn't quite clear. Maybe a docstring to clarify?